docs/reference/supported-environments.md

```diff
@@ -33,7 +33,25 @@ For example, 1.2+ means support for the minor version 2, and any higher minors (
 
 ## Camunda 8 Self-Managed
 
-We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Please follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
+We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
+
+The following list includes links for supported Kubernetes environments and recommended starting config. Generally speaking, the specification depends on your needs and workloads.
+
+- [Amazon EKS](/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/amazon-eks.md)
+  - Instance type: `m6i.xlarge` (4 vCPUs, 16 GiB Memory)
+  - Number of nodes: `4`
+  - Volume type: `SSD gp3`
+    - Including Amazon EBS CSI driver and StorageClass
+- [Microsoft AKS](/self-managed/platform-deployment/helm-kubernetes/platforms/microsoft-aks.md)
+  - Instance type: `Standard_D4as_v4` (4 vCPUs, 16 GiB Memory)
+  - Number of nodes: `4`
+  - Volume type: `Premium SSD/Premium SSD v2`
+- [Google GKE](/self-managed/platform-deployment/helm-kubernetes/platforms/google-gke.md)
+  - Instance type: `n1-standard-4` (4 vCPUs, 15 GB Memory)
+  - Number of nodes: `4`
+  - Volume type: `Performance (SSD) persistent disks`
+- [Red Hat OpenShift](/self-managed/platform-deployment/helm-kubernetes/platforms/redhat-openshift.md) - 4.11.x, 4.12.x, 4.13.x
+<!-- No current documentation available for recommended starting config.-->
```

Comment:
Are there starting config recommendations for Red Hat? Currently, it's the only platform without them, but with specific supported versions.

Reply:
OpenShift does not have associated instance sizes as it could be installed on any infrastructure.

Reply:
I will adjust this so the expectations are clear and it doesn't look like a miss.